group-coordinator/src/main/java/org/apache/kafka/coordinator/group/assignor/GroupSpecImpl.java

```diff
@@ -56,11 +66,25 @@ public SubscriptionType subscriptionType() {
         return subscriptionType;
     }
 
+    /**
+     * @param topicId           The topic Id.
```

Comment:
nit: We usually have a first line before the params. You can take a look at the other javadocs.